nsxt/resource_nsxt_policy_tier1_gateway.go

```diff
@@ -559,10 +557,15 @@ func resourceNsxtPolicyTier1GatewayRead(d *schema.ResourceData, m interface{}) e
 		return handleReadError(d, "Locale Service for T1", id, err)
 	}
 	var services []map[string]interface{}
+	_, shouldSetLS := d.GetOk("locale_service")
 	if len(localeServices) > 0 {
 
+		if isGlobalManager {
```

Comment:
nit: can we move this out of the `if len(localeServices) > 0` block, like L933-938 of nsxt/resource_nsxt_policy_tier0_gateway.go

Reply:
Good point, done